packages/components/src/config/variables.ts

```diff
@@ -88,3 +88,27 @@ export const LINE_HEIGHT = {
     BASE: '1.8',
     TREZOR_ACTION: '37px',
 };
+
+export const COINS = [
```

Comment:
storybook's config vars shouldn't be stored in components lib.

Reply:
I don't think it's a storybook's var, i think it belongs to the components... 
It's good to have a array of all the available coins - so you don't need to define this array in your app and it's on a single place and always up-to-date with version of components.